examples/llama2/llama2_genai.json

```diff
@@ -0,0 +1,56 @@
+{
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "hf_config": {
+                "model_name": "meta-llama/Llama-2-7b-hf",
+                "model_class": "LlamaForCausalLM",
+                "task": "text-generation"
+            }
+        }
+    },
+    "systems": {

```

Comment:
this system is not being used so it's safe to delete?

Reply:
Removed